BeatSaberMarkupLanguage/BeatSaberUI.cs

```diff
@@ -143,9 +143,10 @@ public static TMP_FontAsset CreateTMPFont(Font font, string nameOverride = null)
         /// <param name="text">The text to be displayed.</param>
         /// <param name="anchoredPosition">The position the button should be anchored to.</param>
         /// <returns>The newly created TextMeshProUGUI component.</returns>
+        /// <remarks>Proxied to the generic method, but kept for binary compatibility</remarks>
         public static TextMeshProUGUI CreateText(RectTransform parent, string text, Vector2 anchoredPosition)
         {
-            return CreateText(parent, text, anchoredPosition, new Vector2(60f, 10f));
+            return CreateText<CurvedTextMeshPro>(parent, text, anchoredPosition, new Vector2(60f, 10f));

```

Comment:
I wasn't sure whether this should have been the generic method with 3 parameters or the one with 4.
Currently calling the one with 4 parameters as it prevents a single call in the call stack when being called.

Let me know if this has to be changed.